README.md

```diff
@@ -103,6 +104,14 @@ Optional
 
 > :warning: Anytime you edit the `customer.env` file to change the CARTO configuration you will need to run the `install.sh` script to updathe the `.env` file used by Docker compose
 
+#### Custom Service Account
+
+For each CARTO self hosted installation, we create and provide a Google Cloud Platform Service Account key that the CARTO backend uses to authenticate requests to Google APIs. 
```

Comment:
let's add -> `CARTO deploys a dedicated infrastructure for every self hosted installation and such Service Account key is required to use some of the services deployed`.